Makefile

```diff
@@ -1,3 +1,3 @@
 test:
 	@echo "  >  Running unit tests"
-	go test ./...
+	go test --race ./...
```

Comment:
please create a new entry called `test-with-race` and leave the old `test` as it was

Reply:
done